portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,11 +68,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String comment = getParameter(request, "text-input", "");
+    String name = getParameter(request, "name-input", "");
+    String message = getParameter(request, "text-input", "");
+    long timestamp = System.currentTimeMillis();
+
+    Date date = new Date(timestamp);
+    SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
+    String time = dateFormat.format(date);
+
+    Entity commentEntity = new Entity(Comment.MESSAGE_KEY);
+    commentEntity.setProperty(Comment.NAME_KEY, name);
+    commentEntity.setProperty(Comment.MESSAGE_KEY, message);
+    commentEntity.setProperty(Comment.TIMESTAMP_KEY, timestamp);
+    commentEntity.setProperty(Comment.TIME_KEY, time);
     
-    comments.add(comment);
-    response.setContentType("text/html;");
-    response.getWriter().println(comments);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+    //comments.add(comment);
+    //response.setContentType("text/html;");
+    //response.getWriter().println(comments);
```

Comment:
No need to keep the comments in your source code when the comments are not necessary.